src/ui/components/List/ListView.vue

```diff
@@ -155,6 +157,10 @@ export default {
           })
         );
       }
+    },
+    itemSelected(item, event) {
+      event.stopPropagation();
+      this.$emit('itemSelectionChanged', item, event.currentTarget);
```

Comment:
Emit names should be in `kebab-case` per convention
```suggestion
      this.$emit('item-selection-changed', item, event.currentTarget);
```